definitions/push-images/skopeo-copy-task.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: skopeo-copy-task
+spec:
+  params:
+    - name: srcImageURL
+      description: Source URL
+      type: string
+    - name: destImageURL
+      description: Destination URL
+      type: string
+    - name: srcToken
+      description: Robot bearer token for source
+      type: string
+    - name: destToken
+      description: Robot bearer token for destination
+      type: string
+    - name: retries
+      description: Retry skopeo copy N times.
+      type: int
+      default: 0
+  steps:
+    - name: skopeo-copy
+      image: quay.io/skopeo/stable:v1.7.0
+      # yamllint disable rule:line-length
```

Comment:
Longest line you have here is 97 characters long. The limit as per your other PR is 120, so you don't need this isn't it?

Reply:
fixed